providers/dns/azure/azure.go

```diff
@@ -169,9 +170,9 @@ func (d *DNSProvider) Present(domain, token, keyAuth string) error {
 	ctx := context.Background()
 	fqdn, value := dns01.GetRecord(domain, keyAuth)
 
-	zone, err := d.getHostedZoneID(ctx, fqdn)
-	if err != nil {
-		return fmt.Errorf("azure: %w", err)
+    zone, err := d.getHostedZoneID(ctx, fqdn)
+    if err != nil {
+            return fmt.Errorf("azure: %w", err)
```

Comment:
In Go code, we do want tabs, not spaces :-)

Please run `gofmt -s -w providers/dns/azure/azure.go` to correct this automatically.